### ISSUE TITLE
[BUG] `v-img` incorrectly reports `position` as deprecated

### ISSUE THREAD

**Issue by K-kind** [vuetifyjs/eslint-plugin-vuetify#83]
When I try to use the position props for v-img, I get this message (vuetify 3.4.9, eslint-plugin-vueitfy 2.1.1):

```
error  'position' has been removed
```

However, it is listed in the [documentation](https://vuetifyjs.com/en/api/v-img/#props-position) and it does indeed work as it did in Vuetify 2.

**Example**

<img width="693" alt="demo" src="https://github.com/vuetifyjs/eslint-plugin-vuetify/assets/55728594/cd4d52f0-ccab-4f18-b8e9-3d1d0034458e">

```vue
<v-card width="200" height="200">
  <v-img
    src="https://cdn.vuetifyjs.com/images/parallax/material.jpg"
    position="left"
    height="200"
    cover
  />
</v-card>
```

**Comment by ricardo17coelho** [comment]
I'm also getting this message too.
